server/api/models/booking.js

```diff
@@ -1,105 +1,89 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable camelcase */
+
+import db from '../db/Db';
+
 class Booking {
-	constructor() {
-		this.bookings = [{
-			booking_id: 1,
-			trip_id: 1,
-			user_id: 1,
-			bus_license_number: 'KGA 344R',
-			trip_date: '21/09/2019',
-			first_name: 'John',
-			last_name: 'Doe',
-			email: 'johndoe@gmail.com',
-			seat_number: 23,
-		},
-		{
-			booking_id: 2,
-			trip_id: 1,
-			user_id: 1,
-			bus_license_number: 'KGA 344R',
-			trip_date: '21/10/2019',
-			first_name: 'John',
-			last_name: 'Doe',
-			email: 'maryjane@gmail.com',
-			seat_number: 24,
-		},
-		{
-			booking_id: 3,
-			trip_id: 1,
-			user_id: 2,
-			bus_license_number: 'KGA 344R',
-			trip_date: '21/11/2019',
-			first_name: 'Test',
-			last_name: 'User',
-			email: 'testuser@gmail.com',
-			seat_number: 34,
-		},
-		{
-			booking_id: 4,
-			trip_id: 1,
-			user_id: 2,
-			bus_license_number: 'KTZ 590M',
-			trip_date: '21/09/2019',
-			first_name: 'Jane',
-			last_name: 'Doe',
-			email: 'janedoe@gmail.com',
-			seat_number: 4,
-		},
-		];
-	}
+	// eslint-disable-next-line no-useless-constructor
+	constructor() {}
 
-	createNewBooking({
+	// eslint-disable-next-line class-methods-use-this
```

Comment:
Add it in the eslint configuration's rule instead

Reply:
Added to the eslint config file